### ISSUE TITLE
Position of audiohandle doesn't work

### ISSUE THREAD

**Issue by CamperPR** [underscorediscovery/luxe#396]
There is example:
```haxe
var handle = Luxe.audio.play(source, volume);

override function update (dt:Float){
     trace(Luxe.audio.position_of(handle)); // so, it always outputs: 0
}
```
And what it must output?

**Comment by underscorediscovery** [comment]
What are you testing on?
How long is the sound?

I put this in `tests/features/audio`:

```haxe
    override function update(dt:Float) {

        if(Luxe.audio.state_of(last_played) == luxe.Audio.AudioState.as_playing) {
            trace(Luxe.audio.position_of(last_played));
        }

    }
```

And it works as expected on native.
[On web it will return 0](https://github.com/underscorediscovery/snow/blob/master/snow/modules/webaudio/Audio.hx#L405-L414)

The reason is largely because [web audio api doesn't support this concept](https://github.com/WebAudio/web-audio-api/issues/296) (yet, maybe).

**Comment by CamperPR** [comment]
Yes, I worked on Web target. It's not very good news for me, but I'll wait for changes.

**Comment by underscorediscovery** [comment]
Not much I can do about browsers not supporting these basic features I'm afraid! 
I'll see if we can work around it but it's probably not going to be a best fit until the browsers finish the API. For streaming sounds I think the position is knowable already, which I'll add soon.
